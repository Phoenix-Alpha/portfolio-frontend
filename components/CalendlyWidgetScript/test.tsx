import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import { CalendlyWidgetScript } from '.'

describe('<CalendlyWidgetScript />', () => {
  it('should renders correct <CalendlyWidgetScript />', () => {
    const wrapper = shallow(<CalendlyWidgetScript />)

    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})
