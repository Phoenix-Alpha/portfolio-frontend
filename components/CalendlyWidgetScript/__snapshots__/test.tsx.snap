// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CalendlyWidgetScript /> should renders correct <CalendlyWidgetScript /> 1`] = `
<script
  async={true}
  src="https://assets.calendly.com/assets/external/widget.js"
  type="text/javascript"
/>
`;
